### DISCUSSION TITLE
how to flush firewall rules added by vpc cni in Amazon Linux distros (bottlerocket)?

### DISCUSSION THREAD

**Discussion by tirelibirefe** [cilium/cilium#36796] [Q&A] [answered]
Hello,
Regardding to the [documentation](https://docs.cilium.io/en/latest/installation/k8s-install-helm/#install-cilium) , iptables rules added by VPC CNI must be flushed for overlay mode.

iptables -t nat -F AWS-SNAT-CHAIN-0 \\
   && iptables -t nat -F AWS-SNAT-CHAIN-1 \\
   && iptables -t nat -F AWS-CONNMARK-CHAIN-0 \\
   && iptables -t nat -F AWS-CONNMARK-CHAIN-1

...but these commands are not valid for Amazon bottlerocket worker nodes. What are the correct/valid commands to flush firewall rules added by VPC CNI?

Thanks&Regards

**Comment by joestringer** [comment, marked as answer]
You might consider deploying a common distribution container like Ubuntu (with super user privileges) to run these commands if the host Linux distribution is lacking these tools.
